samples/charts/category-chart/marker-options/Pages/CategoryChartMarkerOptions.razor

```diff
@@ -0,0 +1,191 @@
+                           YAxisMinimumValue="0">
+            </CategoryChart>
+
+            @*MarkerTypes="@MarkerTypes"*@
+        }
+
+    </div>
+</div>
+
+@code {
+
+    private List<OlympicMedals> DataSource;
+    private CategoryChartType ChartType;
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        CategoryChartModule.Register(InfragisticsBlazor);
+
+        var olympicMedals = new List<OlympicMedals>();
+        olympicMedals.Add(new OlympicMedals() { Year = "1996", USA = 148, CHN = 110, RUS = 95 });
+        olympicMedals.Add(new OlympicMedals() { Year = "2000", USA = 142, CHN = 115, RUS = 91 });
+        olympicMedals.Add(new OlympicMedals() { Year = "2004", USA = 134, CHN = 121, RUS = 86 });
+        olympicMedals.Add(new OlympicMedals() { Year = "2008", USA = 131, CHN = 129, RUS = 65 });
+        olympicMedals.Add(new OlympicMedals() { Year = "2012", USA = 135, CHN = 115, RUS = 77 });
+        olympicMedals.Add(new OlympicMedals() { Year = "2016", USA = 146, CHN = 112, RUS = 88 });
+
+        DataSource = olympicMedals;
+
+        this.ChartType = CategoryChartType.Auto;
+    }
+
+    private void OnChartTypeChanged(ChangeEventArgs args)
+    {
+        string str = args.Value.ToString();
+        switch (str)
+        {
+            case "Automatic":
+                {
+                    ChartType = CategoryChartType.Auto;
+                    break;
+                }
+            case "Area":
+                {
+                    ChartType = CategoryChartType.Area;
+                    break;
+                }
+            case "Column":
+                {
+                    ChartType = CategoryChartType.Column;
+                    break;
+                }
+            case "Point":
+                {
+                    ChartType = CategoryChartType.Point;
+                    break;
+                }
+            case "Line":
+                {
+                    ChartType = CategoryChartType.Line;
+                    break;
+                }
+            case "Spline":
+                {
+                    ChartType = CategoryChartType.Spline;
+                    break;
+                }
+            case "SplineArea":
+                {
+                    ChartType = CategoryChartType.SplineArea;
+                    break;
+                }
+            case "StepArea":
+                {
+                    ChartType = CategoryChartType.StepArea;
+                    break;
+                }
+            case "StepLine":
+                {
+                    ChartType = CategoryChartType.StepLine;
+                    break;
+                }
+            case "Waterfall":
+                {
+                    ChartType = CategoryChartType.Waterfall;
+                    break;
+                }
+        }
+    }
+
+    private void OnMarkerTypeChanged(ChangeEventArgs args)
+    {
+        string str = args.Value.ToString();
+        switch (str)
```

Comment:
this `switch` block is not needed because you can do this:
// TODO un-comment when MarkerType is exposed in Category Chart
```this.MarkerType = (MarkerType)args.Value;```